src/main/scala/com/research/nomad/markii/dataflow/AbstractValuePropVASCO.scala

```diff
@@ -379,6 +379,20 @@ class AbstractValuePropVASCO(entryPoints: List[SootMethod])
             }
             println("Unhandled: " + invokeExpr.getMethod.getSignature)
           }
+          if (invokeExpr.getMethod.getName == "setTitle") {
+            if(invokeExpr.getMethod.getSignature == "<androidx.appcompat.app.AlertDialog: void setTitle(java.lang.CharSequence)>") {
```

Comment:
I remember that in Android there are many packages for Dialog due to compatibility problems. `<androidx.appcompat.` is only one of them. Check this: https://github.com/izgzhen/markii/blob/8657cfe02c8436f4f1193df0d28a3b7f41d7b9fa/src/main/scala/com/research/nomad/markii/Constants.scala#L70 and maybe reuse the constant defined here.